docs/tutorials/Qiskit Algorithms Transition Guide.ipynb

```diff
@@ -0,0 +1,1629 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Qiskit Algorithms Transition Guide\n",
```

Comment:
Do we want to use the term `Migration` here rather than `Transition` for consistency since we are calling the docs here a Migration Guide and this part is for the Algorithms

Reply:
That is a fair comment. This notebook is an incomplete version of Julien's just for testing. I asked him if he could commit it here directly when he finishes to keep his name on it and then he or I will change the title. Otherwise, I will change the title, merge the PR and he can put his final version on another PR.

Reply:
I changed the file name ant title to MIgration